pom.xml

```diff
@@ -206,6 +206,12 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```

Comment:
What is ultimately bringing in and using this dependency?

Reply:
Right, but my question is what is using it?

Reply:
It is being transitively used by the modules presto-hive-hadoop2 , presto-hive , presto-native-execution and presto-hive-metastore . (Attaching a screenshot for reference)
<img width="838" alt="Screenshot 2024-12-04 at 9 43 18 PM" src="https://github.com/user-attachments/assets/d39c0c68-40bb-488d-9a4f-6037fdb14467">

Reply:
Should we update hive-dwrf instead of overriding it at the top level here?

Reply:
@tdcmeehan , introducing the latest version of hive-dwrf introduces a lot of vulnerabilities.(https://mvnrepository.com/artifact/com.facebook.hive/hive-dwrf/0.18.9). (Attaching the screenshot from the maven repository). 
<img width="1236" alt="Screenshot 2024-12-05 at 8 49 08 PM" src="https://github.com/user-attachments/assets/ffd24d48-f438-4c31-9e52-20eee64cdb6a">

Reply:
@tdcmeehan , could you please have a look at the comments and re-review if possible?

Reply:
What I mean is, simply fix the vulnerabilities in hive-dwrf, cut a release, then update here. https://github.com/prestodb/presto-hive-dwrf